google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java

```diff
@@ -781,7 +785,12 @@ static ListLogEntriesRequest listLogEntriesRequest(
     String filter = FILTER.get(options);
     if (filter != null) {
       builder.setFilter(filter);
+    } else {
+      // If filter is not specified, default filter is looking back 24 hours in line with gcloud
+      // behavior
+      builder.setFilter(createDefaultTimeRangeFilter());
```

Comment:
The way I did it in Python (and gcloud does it) is I check if the filter has "timezone" as a substring, and if not I'd add the default timezone filter

Reply:
Yeah, I just did it too now.